scripts/gh-review-threads.sh

```
@@ -0,0 +1,443 @@
+    ;;
+
+  list-unresolved-ndjson)
+    threads=$(fetch_threads)
+    render_unresolved_ndjson "$threads"
+    ;;
+
+  resolve-all-unresolved)
+    info "Fetching threads…"
+    threads=$(fetch_threads)
+    readarray_compat tids bash -lc 'jq -r '\''.[] | select(.isResolved == false) | .id'\'' <<<"$threads"'
+    if [[ ${#tids[@]} -eq 0 ]]; then
```

Shellcheck (0.10.0)

[warning] tids is referenced but not assigned.

(SC2154)